extensions/smallrye-opentracing/runtime/src/main/java/io/quarkus/smallrye/opentracing/runtime/graal/Target_org_jboss_resteasy_microprofile_config_FilterConfigSource.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.smallrye.opentracing.runtime.graal;
+
+import org.jboss.resteasy.microprofile.config.FilterConfigSource;
```

Comment:
I'm not particularly excited to have RESTEasy related substitutions in the opentracing extension?

Shouldn't this be part of the RESTEasy extension itself?

Reply:
It wouldn't hurt, but it's not really necessary because the problem only surfaces because of how OpenTracing relies on some servlet stuff being available.
IMHO, we should leave it here, but keep it mind for the future (but I seriously doubt it would ever be needed because we would have long have run into the problem).

Reply:
I suppose I'll keep my "I told you so" for later :).

Reply:
Well you know that if it happens, I'll fix it ASAP :)